### PR TITLE
Rename --version or --resnet_version

### PR DESCRIPTION
**Pull request by robieta**
Inside Google's testing infrastructure the namespace `version` is already taken, and this is breaking testing.

### CODE REVIEW

File: official/resnet/resnet_run_loop.py [hunk 1]
```diff
@@ -379,7 +379,7 @@ def resnet_main(
           'resnet_size': int(flags_obj.resnet_size),
           'data_format': flags_obj.data_format,
           'batch_size': flags_obj.batch_size,
-          'version': int(flags_obj.version),
+          'version': int(flags_obj.resnet_version),
```
Comment (karmel) [inline official/resnet/resnet_run_loop.py]: I feel like we should update throughout, to avoid confusion.

File: official/resnet/imagenet_test.py [hunk 2]
```diff
@@ -280,25 +280,25 @@ def test_imagenet_end_to_end_synthetic_v2(self):
...
         main=imagenet_main.run_imagenet, tmp_root=self.get_temp_dir(),
-        extra_flags=['-v', '1', '-rs', '200']
+        extra_flags=['-resnet_version', '1', '-resnet_size', '200']
     )
 
   def test_imagenet_end_to_end_synthetic_v2_huge(self):
     integration.run_synthetic(
         main=imagenet_main.run_imagenet, tmp_root=self.get_temp_dir(),
-        extra_flags=['-v', '2', '-rs', '200']
+        extra_flags=['-resnet_version', '2', '-resnet_size', '200']
```
Comment (karmel) [inline official/resnet/imagenet_test.py]: I consider this a victory for fully_specified_arg_names.

Comment (robieta) [reply official/resnet/imagenet_test.py]: yeah yeah

**Comment by robieta** [comment]
@karmel renamed throughout.

File: official/resnet/layer_test.py [hunk 3]
```diff
@@ -41,14 +41,22 @@
 DATA_FORMAT = "channels_last"  # CPU instructions often preclude channels_first
 BATCH_SIZE = 32
 BLOCK_TESTS = [
-    dict(bottleneck=True, projection=True, version=1, width=8, channels=4),
```
Comment (robieta) [inline official/resnet/layer_test.py]: @karmel Behold the casualties of your verbosity!

File: official/resnet/imagenet_main.py [hunk 4]
```diff
@@ -207,8 +207,8 @@ def __init__(self, resnet_size, data_format=None, num_classes=_NUM_CLASSES,
         data format to use when setting up the model.
       num_classes: The number of output classes needed from the model. This
         enables users to extend the same model to their own datasets.
-      version: Integer representing which version of the ResNet network to use.
-        See README for details. Valid values: [1, 2]
+      resnet_version: Integer representing which version of the ResNet network
+      to use. See README for details. Valid values: [1, 2]
```
Comment (karmel) [inline official/resnet/imagenet_main.py]: nit: missing indent.

File: official/resnet/layer_test.py (hunk 3 above)
Comment (karmel) [reply official/resnet/layer_test.py]: I shed a tear for the fallen.

**Pull request merged**